### DISCUSSION TITLE
How long does Elementor team expect the Improved Asset Loading & Optimized DOM Output experiments to last?

### DISCUSSION THREAD

**Discussion by d19dotca** [elementor/elementor#14433] [Developers Community]
I'm curious when we'll be moving them from Alpha and Beta to full-production? Are we looking at months yet, or closer to a year? Just seems like Improved Asset Loading and Optimized DOM Output are fairly stable (especially when not using third-party Elementor plugins), so it's surprising to Improved Asset Loading in Alpha stage still and not beta for example. And thus the Optimized DOM Output which started a very long time ago, to see it still in Beta is odd. Just my observations though, I'm guessing they're still in those states and "experiments" because they're not stable enough yet overall?

**Comment by nicholaszein** [comment]
Hey @d19dotca!

Both features have been merged to core as of version 3.19 for the **Optimized DOM Output**, and version 3.21 for the **Improved Asset Loading**.

Cheers! 🥂
